### DISCUSSION TITLE
vue how to use typst

### DISCUSSION THREAD

**Discussion by 315574925** [typst/typst#3709] [Q&A]
vue how to use typst

**Comment by emilyyyylime** [comment]
We can not answer a question with no details. Please explain your circumstances and what you're trying to achieve

**Reply by 315574925** [reply]
In the vue project, the writing function is implemented with typst, which is similar to the function of word document

**Reply by emilyyyylime** [reply]
Sorry, no one can help you if you're this vague. This is not google

**Reply by 315574925** [reply]
我应用场景很简单，就是想在我的vue项目中集成typst，实现在线写作功能。

**Reply by Myriad-Dreamin** [reply]
Check it https://github.com/mitex-rs/mitex/blob/main/packages/mitex-web/src/tools/underleaf.html

**Reply by Myriad-Dreamin** [reply]
The underleaf uses typst, which is deployed here. https://mitex-rs.github.io/mitex/tools/underleaf.html

![image](https://github.com/typst/typst/assets/35292584/f48a336b-7b96-42b9-ad61-2686256367ce)

**Comment by frozolotl** [comment]
You can check out [typst.ts](https://github.com/Myriad-Dreamin/typst.ts).
It is not an ideal solution, but that won't be available until Typst supports actual HTML export.
